content/docs/SDKs/javascript/meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "changelog": "JavaScript SDK Changelog",
+  "vercel-ai-sdk": "Vercel AI SDK"
+}
```

Comment:
remove this file.

@rblalock should we exclude this change from the prompt somehow?